.github/workflows/ci.yml

```diff
@@ -39,6 +39,8 @@ jobs:
         args: -v --build-tags relic
         # https://github.com/golangci/golangci-lint-action/issues/244
         skip-pkg-cache: true
+        # Make sure we get accurate results
+        # Some of these may be needed if observer code is integrated: only-new-issues: true, skip-build-cache: true
```

Comment:
I think we might want to add these back, in case we run into false positive lint issues again.
I can remove it but it saves time to developers.

Reply:
why are those setting required for observers, but not any other type?

Reply:
I think it was an infrastructure issue within GitHub. I have not seen it in a while.
It posted a few false positive Lint issues.
The reason I would like to leave the comment here, so that new team members can notice and quickly fix, what took me significant effort. The issue is known, intermittent and not yet resolved.